superset-frontend/src/components/Datasource/CollectionTable.tsx

```diff
@@ -102,9 +101,28 @@ const CrudTableWrapper = styled.div<{ stickyHeader?: boolean }>`
         min
       }
     `}
-  th span {
-    vertical-align: ${({ theme }) => theme.gridUnit * -2}px;
-  }
+  ${({ theme }) => `
+    th span {
+      vertical-align: ${theme.gridUnit * -2}px;
+    }
+    .text-right {
+      text-align: right;
+    }
+    .empty-collection {
+      padding: ${theme.gridUnit * 2 + 2}px;
```

Comment:
I noticed that you removed the `.control-label` and `.frame` classes. It was intended?

Reply:
@michael-s-molina yes I checked and those two are unused

Reply:
No. I just wanted to confirm that it was removed intentionally.